docs/src/main/docs/microprofile/06_configuration.adoc

```diff
@@ -144,7 +144,7 @@ This approach is based on a pair of keys: a public key which is known to anybody
 - a value encrypted by a public key can only be decrypted by the private key
 - a value encrypted by a private key can only be decrypted by the public key
 
-When using the config encryption filter, you should encrypt the configuration values
+When using the config encryption filter, you have to encrypt the configuration values
```

Comment:
why? We provide recommendations - there is no enforcement to encrypt, unless the value is marked as a secret

Reply:
I meant  it like that you cant select between public and private. That you have to use public for encryption.